src/OpenApi/test/OpenApiGeneratorTests.cs

```diff
@@ -228,6 +229,69 @@ public void AddsMultipleResponseFormatsFromMetadataWithIResult()
         Assert.Empty(badRequestResponseType.Content);
     }
 
+    [Fact]
+    public void DefaultResponseDescriptionIsCorrect()
+    {
+        var operation = GetOpenApiOperation(
+        [ProducesResponseType(typeof(TimeSpan), StatusCodes.Status201Created)]
+        [ProducesResponseType(StatusCodes.Status400BadRequest)]
+        () => new InferredJsonClass());
+
+        Assert.Equal(2, operation.Responses.Count);
+
+        var successResponse = operation.Responses["201"];
+        Assert.Equal("success", successResponse.Description);
+
+        var clientErrorResponse = operation.Responses["400"];
+        Assert.Equal("client error", clientErrorResponse.Description);
+    }
+
+    [Fact]
+    public void DefaultResponseDescriptionIsCorrectForTwoSimilarResponses()
+    {
+        var operation = GetOpenApiOperation(
+        [ProducesResponseType(StatusCodes.Status100Continue)]
+        [ProducesResponseType(StatusCodes.Status101SwitchingProtocols)]
+        () => new InferredJsonClass());
+
+        Assert.Equal(2, operation.Responses.Count);
+
+        var continueResponse = operation.Responses["100"];
+        Assert.Equal("information", continueResponse.Description);
+
+        var switchingProtocolsResponse = operation.Responses["101"];
+        Assert.Equal("information", switchingProtocolsResponse.Description);
+    }
+
+    [Fact]
+    public void AllDefaultResponseDescriptions()
+    {
+        var operation = GetOpenApiOperation(
+        [ProducesResponseType(StatusCodes.Status100Continue)]
```

Comment:
Should cover cases like `[ProducesResponseType(42)]` and `[ProducesResponseType(654)]`

Reply:
Added another test for this.